### PR TITLE
Change contrast  and remove inline css in navigation button

### LINKED ISSUE #2897: VPAT -- Contrast Errors in Gradeable Forms
Per WAVE:
> **Very Low Contrast**
> **What It Means**
> Very low contrast between foreground and background colors.
> **Why It Matters**
> Adequate contrast is necessary for all users, especially users with low vision.
> **How to Fix It**
> Increase the contrast between the foreground (text) color and the background color. Large text (larger than 18 point or 14 point bold) does not require as much contrast as smaller text.
> **The Algorithm... in English**
> Text is present that has a contrast ratio less than 4.5:1, or large text (larger than 18 point or 14 point bold) has a contrast ratio less than 3:1. Elements with background images must have a background color defined that provides adequate contrast when images are disabled or unavailable.
> **Standards and Guidelines**
> [1.4.3 Contrast (Minimum) (Level AA)](http://webaim.org/standards/wcag/checklist#sc1.4.3)

There are several contrast errors in the Gradeable forms that need attention.
The top of the form has a row of buttons to denote workflow:

- "grayed out" disabled buttons have a very light gray background (`#f0f0f0`) and a white font (`#ffffff`).  Contrast ratio is 1.14:1
- "grayed out" active buttons used for navigation through a workflow have a medium gray background (`#999999`) and a white font (`#ffffff`).  Contrast ratio is 2.85:1

In the form:

- "Required" tags in the form are on a white background (`#ffffff`) with a red font (`#ff0000`).  Contrast ratio is 4:1 (needs to be 4.5:1 or better).

### PR DESCRIPTION
**Pull request by drumilpatel2000**
Fixes #2897 
With reference #2271 Inline CSS is removed in admin gradeable.
screenshot from 2019-02-15 23-19-51
Disabled button Is made visible by using a dark foreground and more dark text color 
screenshot from 2019-02-15 23-20-29
Normal Button(That can be pressed) has outline which is removed on hovering this will give feeling of being pressed

### CODE REVIEW

**Comment by drumilpatel2000** [comment]
@pbailie please review

**Comment by drumilpatel2000** [comment]
@pbailie Please review at this

**Comment by pbailie** [comment]
Latest proposed UI color scheme in this PR is of sufficient contrast ratio.  This will satisfy fixing #2897 (VPAT -- Contrast Errors in Gradeable Forms).

@bmcutler As I do not have authority to merge in pull requests, please further review this PR.

**Comment by drumilpatel2000** [comment]
Thanks @pbailie

File: site/public/css/admin-gradeable.css [hunk 1]
```diff
@@ -122,3 +122,24 @@ div.btn-container {
...
+}
+.normalBtn:hover{
+    border: none;
+}
+
+.activeBtn{
+    background-color: #008000;
+    color: #ffffff;
+    border: none;
+}
```
Comment (MasterOdin) [inline site/public/css/admin-gradeable.css]: rename these to be `disabled-btn`, `normal-btn`, and `active-btn` please to match the naming scheme of the other CSS classes.

Comment (drumilpatel2000) [reply site/public/css/admin-gradeable.css]: Thanks for helping @MasterOdin .
I have changed all changes that you requested

Comment (drumilpatel2000) [reply site/public/css/admin-gradeable.css]: @pbailie  can you please review my code again

**Comment by drumilpatel2000** [comment]
@pbailie Can you please review this code again

**Comment by bmcutler** [comment]
thanks for your contribution!  :)

**Pull request merged**